doc/api_methods.md

```diff
@@ -114,20 +130,62 @@ An overlay is internally a [Backbone.View](http://backbonejs.org/#View) so if yo
 
 ### vis.addInfowindow(_map, layer, fields [, options]_)
 
-Adds an infowindow to the map controlled by layer events. It enables interaction and overrides the layer interactivity.
+Infowindows provide additional interactivity for your published map, controlled by layer events. It enables interaction and overrides the layer interactivity. A pop-up information window appears when a viewer clicks, or hovers their mouse over, select data on your map. 
+
+**Note:** By default, the `vis.addInfowindow` code displays interactivity for the infowindow with the "click" action. If you are using the "hover" action, you will still have to define the infowindow options with `viz.addInfowindow`, and specify additional parameters for the hover action with the `vis.Overlays` code.
 
 #### Arguments
 
 Option | Description
 --- | ---
-map | native map object or leaflet
-layer | cartodb layer (or sublayer)
-fields | array of column names
+map | native map object or leaflet.
+infowindowTemplate | the script type and id defined in your infowindow_template.
+templateType | the infowindow type defined with [Mustache template](http://mustache.github.io/mustache.5.html) placeholders.
+click | By default, the "click" action is used when infowindow interactivity is enabled with the `vis.addInfowindow` code.
+hover | If you want the infowindow to appear with the "hover" action, you must define additional hover parameters with the `vis.addOverlay` code:
+options |
+--- | ---
+&#124;_ type | defines the `vis.addOverlay` [option](#visaddoverlayoptions) as infowindow.
+&#124;_ layer | cartodb layer (or sublayer).
+&#124;_ template | the tooltip content wrapper for the hover template.
+&#124;_ position | defines the position of the hover action.
+&#124;_ fields | array of column names.
+
+**REVIEWER**: WHAT ARE REQUIRED OPTIONS? ALSO, WHAT ARE SOME OF THE OTHER POSSIBLE VALUES?
+
+**Tip:** See [How can I use CartoDB.js to create and style infowindows?](/faqs/infowindows/#how-can-i-use-cartodb.js-to-create-and-style-infowindows) for an overview of how to create infowindows.
 
 #### Returns
 
 An infowindow object, see [sublayer.infowindow](#sublayerinfowindow)
 
+#### Example
+
+The following example displays how to enable infowindow interactivity with the "click" action.
+
+{% highlight html %}
+ cartodb.vis.Vis.addInfowindow(map, sublayer, ['cartodb_id', 'lat', 'lon', 'name'],{
+           infowindowTemplate: $('#infowindow_template').html(),
+           templateType: 'mustache'
+         });
+{% endhighlight %}
+
+**Tip:** See [`vis.addOverlay`](#visaddoverlayoptions) for how to define a hover action for an infowindow.
+
+#### Example (Infowindow with Hover Action)
```

Comment:
IMO, this example doesn't make too much sense for me since this is the section about `cartodb.vis.Vis.addInfowindow`.